src/components/icon/examples/icon-image.tsx

```diff
@@ -0,0 +1,148 @@
+import { Component, h } from '@stencil/core';
+
+/**
+ * Displaying an image instead of an icon
+ *
+ * Because the `limel-icon` component is used by several other components
+ * already, we have added the ability to display an image instead of an icon.
+ * This makes it possible to display a user avatar, for example, anywhere that
+ * an icon is used.
+ *
+ * To render an image, simply provide the URL to the image as the `name`
+ * property.
+ *
+ * Supported url formats are:
+ * - Absolute URL (beginning with `https://` or `http://`)
+ * - Relative URL (beginning with a single slash)
+ * - Protocol-relative URL (beginning with double slashes)
+ *
```

Comment:
do we need anything regarding file-size, canvas size, etc?

Reply:
Yeah, we might want to add something about file size.

By "canvas size", I assume you mean the image's dimensions? The image is resized to fit the size of `limel-icon`, and border-radius is applied to make it round. I guess we might want to add a line telling people to use square images 👍